test/sample/test_sample.py

```diff
@@ -836,3 +856,90 @@ def test_thermodynamic_integration():
     assert np.isclose(log_evidence, np.log(evidence[0]), atol=tol)
     assert np.isclose(log_evidence_not_all, np.log(evidence[0]), atol=tol)
     assert np.isclose(log_evidence_simps, np.log(evidence[0]), atol=tol)
+    assert np.isclose(
+        log_evidence_steppingstone, np.log(evidence[0]), atol=tol
+    )
+
+
+@pytest.mark.flaky(reruns=2)
+def test_harmonic_mean_log_evidence():
+    tol = 1
+    # define problem
+    problem = gaussian_problem()
+
+    # run optimization and MCMC
+    result = optimize.minimize(
+        problem,
+        progress_bar=False,
+    )
+    result = sample.sample(
+        problem,
+        n_samples=2000,
+        result=result,
+    )
+
+    # compute the log evidence using harmonic mean
+    harmonic_evidence = sample.util.harmonic_mean_log_evidence(result)
+    # compute the log evidence using stabilized harmonic mean
+    prior_samples = np.random.uniform(problem.lb, problem.ub, size=100)
+    harmonic_stabilized_evidence = sample.util.harmonic_mean_log_evidence(
+        result=result,
+        prior_samples=prior_samples,
+        neg_log_likelihood_fun=problem.objective,
+    )
+
+    # compute real evidence
+    evidence = quad(
+        lambda x: 1
+        / (problem.ub[0] - problem.lb[0])
+        * np.exp(gaussian_llh(x)),
+        a=problem.lb[0],
+        b=problem.ub[0],
+    )
+
+    # compare to known value
+    assert np.isclose(harmonic_evidence, np.log(evidence[0]), atol=tol)
+    assert np.isclose(
+        harmonic_stabilized_evidence, np.log(evidence[0]), atol=tol
+    )
+
+
+@pytest.mark.flaky(reruns=2)
+def test_bridge_sampling():
+    # define problem
+    objective = Objective(
+        fun=lambda x: -gaussian_llh(x),
+        grad=gaussian_nllh_grad,
+        hess=gaussian_nllh_hess,
+    )
+    prior_true = NegLogParameterPriors(
+        [
+            {
+                "index": 0,
+                "density_fun": lambda x: (1 / (10 + 10)),
+                "density_dx": lambda x: 0,
+                "density_ddx": lambda x: 0,
+            },
+        ]
+    )
+    problem = pypesto.Problem(
+        objective=AggregatedObjective([objective, prior_true]),
+        lb=[-10],
+        ub=[10],
+        x_names=["x"],
+    )
+
+    # run optimization and MCMC
+    result = optimize.minimize(
+        problem,
+        progress_bar=False,
```

Comment:
```suggestion
        progress_bar=False,
        n_starts=10
```
Not sure whether you actually need any more than one start but 100 is probably overkill?